content/python/concepts/matplotlib/terms/legend/legend.md

```diff
@@ -0,0 +1,90 @@
+---
+Title: '.legend()'
+Description: 'Places a legend on the axes and access to manipulate parameters of the legend.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Data Visualization'
+Tags:
+  - 'Libraries'
+  - 'Matplotlib'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`.legend()`** function returns a legend on the axes of a plot or subplot.
+
+## Syntax
+
+```pseudo
+plt.legend(handles, labels)
+```
+
+`legend()` can be called without any parameters. This will simply attempt to add the elements automatically. Both `handles` and `labels` are optional. `handles` is utilized if more control is desired with each entry on the plot and how they are labeled. `labels` is a list of string(s) that will display the labels in the order that they are entered with no regard to the actual entries on the plot.
+
+`.legend()` has the following parameters:
```

Comment:
```suggestion
The `.legend()` accepts zero or more parameters and can be called on a valid `plot_instance` of the `pyplot` module. When called with no parameters, the `.legend()` function will attempt to add the elements automatically. The following parameters can be used in the `.legend()` function:
```

A few additional thoughts:

- The sentence "Both `handles` and `labels` are optional." seems confusing to me. If `.legend()` can be called with no parameters, doesn't that mean that _all_ of the parameters mentioned in the table are optional?

- I would remove the following text and work it into the appropriate table rows below: "`handles` is utilized if more control is desired with each entry on the plot and how they are labeled. `labels` is a list of string(s) that will display the labels in the order that they are entered with no regard to the actual entries on the plot."